tsconfig.json

```
@@ -12,6 +12,7 @@
     "esModuleInterop": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["es2020"]
+    "lib": ["es2020"],
+    "sourceMap": true
```

```suggestion
    "sourceMap": true,
    "declaration": true
```

thoughts? so we get `d.ts` files too

---

well, sure